openwisp_monitoring/check/tasks.py

```diff
@@ -57,17 +65,19 @@ def auto_create_ping(model, app_label, object_id):
     if has_check:
         return
     ct = ContentType.objects.get(app_label=app_label, model=model)
-    check = Check(name='Ping', check=ping_path, content_type=ct, object_id=object_id)
+    check = Check(
+        name='Ping', check=ping_path, content_type_id=ct.id, object_id=object_id
```

Comment:
I am not sure why content_type didn't work with `apps.get_model('check', 'Check')` :/
I was getting this strange error, which I couldn't make understand. Anyways, content_type_id works and we achieve the same objective as prior.
```python
  File "/home/hardik/openwisp/openwisp-monitoring/openwisp_monitoring/check/migrations/0003_create_ping.py", line 15, in create_device_ping
    check_model=Check,
  File "/home/hardik/.local/lib/python3.6/site-packages/celery/local.py", line 191, in __call__
    return self._get_current_object()(*a, **kw)
  File "/home/hardik/.local/lib/python3.6/site-packages/celery/app/task.py", line 392, in __call__
    return self.run(*args, **kwargs)
  File "/home/hardik/openwisp/openwisp-monitoring/openwisp_monitoring/check/tasks.py", line 66, in auto_create_ping
    check = Check(name='Ping', check=ping_path, content_type=ct, object_id=object_id)
  File "/home/hardik/.local/lib/python3.6/site-packages/django/db/models/base.py", line 482, in __init__
    _setattr(self, field.name, rel_obj)
  File "/home/hardik/.local/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 219, in __set__
    self.field.remote_field.model._meta.object_name,
ValueError: Cannot assign "<ContentType: config | device>": "Check.content_type" must be a "ContentType" instance.
```